.github/workflows/dynamo_bin_diff.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           dotnet-version: |
             8.0.x
-            10.0.x
```

Comment:
@avidit if I remember correctly, we set the `current` with the desired change, and leave `master` with the previous version we are changing from?